src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -63,10 +63,16 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
           (change)="onConsentChange()"
           name="consent"
         >
-          <span>{{ "ACCEPT" | translate }} </span
-          ><a routerLink=""
+          <span>{{ "ACCEPT" | translate }} </span>
+          <a target="_blank" href=""
             ><span>{{ "CONSENT" | translate }}</span></a
           >
+          <span *ngIf="checkIfLogged() === false">
```

Comment:
usually we should not invoke method in html(), angular invokes them to often and i slows down a page,
try to assign a result to a variable ina a proper place, eg. ngOnInit(), or after receiving it via any rxjs pipline; async pipe also welcomed